styles.css

```diff
@@ -0,0 +1,9 @@
+body {
+    font-family: 'Dosis', sans-serif;
+    color: navy;
+    font-size: 11px;
```

Comment:
The default browser text size is usually 16-18 pixels and you want to stay in that range for readable text. We'll talk more about this as we dive into Project 2 👍🏻